index.js

```diff
@@ -28,17 +28,19 @@ module.exports = {
     return {
       'ember-component-css': {
         namespacing: false
-      },
-      snippetSearchPaths: ['tests/dummy/app'],
-      snippetRegexes: {
-        begin: /{{#docs-snippet\sname=\"(\S+)\"/,
-        end: /{{\/docs-snippet}}/,
-      },
+      }
     };
   },
 
-  included() {
+ included(parentApp) {
     this._super.included.apply(this, arguments);
+    parentApp.options = Object.assign(parentApp.options, {
```

Comment:
This is going to blow away the parent's own `snippetSearchPaths` and `snippetRegexes` options if they happen to have them set. Can we avoid that?